packages/betterer/package.json

```diff
@@ -28,6 +28,7 @@
     "@betterer/logger": "^0.3.0",
     "jest-diff": "^24.9.0",
     "lines-and-columns": "^1.1.6",
+    "safe-string-literal": "^1.0.1",
```

Comment:
Are there any other packages that do a similar thing? 0 stars on Github and 22 downloads a week makes me slightly nervous, but I'm paranoid 😅

Reply:
Not that I saw, but the package is pretty simple: https://github.com/hyurl/safe-string-literal/blob/master/index.js

We can pin the version if you’d like.

Reply:
Technically the meat of the package is just this line: 
```
String(str).replace(/["'`\\\b\f\n\r\t\u2028\u2029]/g
```

Reply:
Let's stay with the dependency I think! Looks like the CI is failing too, you'll need to update the snapshots, then happy to merge!